third_party/kourier-latest/download-kourier.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Download Kourier
+KOURIER_VERSION=0.2.2
+KOURIER_YAML=kourier-knative.yaml
+DOWNLOAD_URL=https://raw.githubusercontent.com/3scale/kourier/v${KOURIER_VERSION}/deploy/${KOURIER_YAML}
+
+wget ${DOWNLOAD_URL}
+
+cat ${KOURIER_YAML} \
+  `# Install Kourier into the kourier-system namespace` \
+  | sed 's/namespace: knative-serving/namespace: kourier-system/' \
```

Comment:
I think we'll also need to add the namespace in the yaml to make this work:
```
apiVersion: v1
kind: Namespace
metadata:
  name: kourier-system
---
```

Reply:
Yep - I missed that when converting things to use the download-kourier.sh script. Added now